commands/activation.go

```diff
@@ -25,7 +25,7 @@ import (
     "time"
 
     "github.com/openwhisk/openwhisk-client-go/whisk"
-    "github.com/openwhisk/openwhisk-cli/wski18n"
+    "../wski18n"
```

Comment:
Please change it back to use the absolute path.

Since the go cli becomes an independent project, we need to structure it according to: https://golang.org/doc/code.html. You need to configure your $GOPATH, and $GOROOT in your local environment, and download openwhisk-cli to the specific path. For example, I put my openwhisk-cli in /Users//go/src/github.com/openwhisk/, and my $GOPATH is /Users//go.

There reason why we need to use the absolute path can be found via the link: https://golang.org/cmd/go/#hdr-Relative_import_paths.